Gamma_Analysis.py

```diff
@@ -0,0 +1,339 @@
+"""
+This will perform a gamma analysis
+It will collect two spectra:
+M = Measurement Spectrum
+B = Background Spectrum
+"""
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import SPEFile
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import os
+# exec('Isotope_identification.py')
+# Use bottom line for Python 3 only. Use top comment for Python 2.7 only.
+exec(open('Isotope_identification.py').read())
+
+
+EFFICIENCY_CAL_COEFFS = [5.1164, 161.65, 3952.3, 30908]
+
+
+def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
+    """
+    Returns absolute efficiencies for a given set of energies, based on a
+    provided efficiency calibration.
+    """
```

Comment:
Also, docstrings should include a description of each input argument.
Here, you want the user to know that `energy` is in keV, and `coeffs` is a list of `[c0, c1, c2, c3]` or however you want to describe it in your plain text equation.